Position.elm

```diff
@@ -1,125 +1,145 @@
 module Position
-  exposing
-    ( Model
-    , Msg
-    , init
-    , initWithLocation
-    , subscriptions
-    , update
-    , view
-    , Role(..)
-    , PositionType(..)
-    , renderEmptySquare
-    )
+    exposing
+        ( Model
+        , Msg
+        , init
+        , initWithLocation
+        , subscriptions
+        , update
+        , view
+        , Role(..)
+        , PositionType(..)
+        , renderEmptySquare
+        )
 
 import Html exposing (Html, div, span)
-import Html.Attributes exposing (style)
+
+
+--import Html.Attributes exposing (style)
+
 import Svg exposing (..)
 import Svg.Attributes
-  exposing
-    ( alignmentBaseline
-    , fill
-    , fontSize
-    , height
-    , points
-    , stroke
-    , strokeWidth
-    , textAnchor
-    , version
-    , viewBox
-    , width
-    , x
-    , y
-    )
-import Color exposing (..)
+    exposing
+        ( alignmentBaseline
+        , fill
+        , fontSize
+        , height
+        , points
+        , stroke
+        , strokeWidth
+        , textAnchor
+        , version
+        , viewBox
+        , width
+        , x
+        , y
+        )
+
+
+-- import Color exposing (..)
+
 import Matrix exposing (Location)
 import Time exposing (Time, second)
 
 
 -- CONSTANTS
 
 
+lightBrown : String
 lightBrown =
-  "peru"
+    "peru"
 
 
+darkBrown : String
 darkBrown =
-  "saddlebrown"
+    "saddlebrown"
 
 
+edgeThickness : number
 edgeThickness =
-  3
+    3
 
 
 
 -- MODEL
 
 
 type alias PieceNumber =
-  Int
+    Int
 
 
 type alias Pixels =
-  Float
+    Float
 
 
 type Role
-  = Head
-  | Link
-  | Tail
+    = Head
+    | Link
+    | Tail
 
 
 type PositionType
-  = Perimeter
-  | Grid
-  | Piece Role PieceNumber
+    = Perimeter
+    | Grid
+    | Piece Role PieceNumber
 
 
 type alias Model =
-  { location : Location
-  , sideSize : Pixels
-  , positionType : PositionType
-  }
+    { location : Location
+    , sideSize : Pixels
+    , positionType : PositionType
+    }
 
 
 init : ( Model, Cmd Msg )
 init =
-  let
-    model =
-      { positionType = Piece Head 43
-      , location = ( 1, 1 )
-      , sideSize = 1000
-      }
-  in
-    ( model, Cmd.none )
+    let
+        model =
+            { positionType = Piece Head 43
+            , location = ( 1, 1 )
+            , sideSize = 50
+            }
+    in
+        ( model, Cmd.none )
 
 
 initWithLocation : Location -> ( Model, Cmd Msg )
 initWithLocation location =
-  let
-    ( model, cmd ) =
-      init
-  in
-    ( { model | location = location }, cmd )
+    let
+        ( model, cmd ) =
+            init
+
+        ( locX, locY ) =
+            location
+
+        pieceNumber =
+            (locY * 11) + locX
+    in
```

Comment:
Actually, `pieceNumber` is independent of the coordinate int the board.  `pieceNumber` is actually intended to be the position of the `piece` in a chain.  At this time, that's not very clear because it's more part of the rules of the game than the layout which we haven't really developed yet.

To show how `pieceNumber` is independent of position, it's very likely that we're going to distribute the pieces randomly in the board.  The objective of one game is to move pieces around to order them.

However, to use this as a device to show all the piece numbers, right now I'm fine w/ this.